src/mongo_client.ts

```diff
@@ -774,14 +774,17 @@ export interface MongoOptions
    *
    * If set TLS enabled, equivalent to setting the ssl option.
    *
+   * Spec complicant options: ssl, tls, tlsCAFile, tlsCertificateKeyFile, tlsCertificateKeyFilePassworda, tlsAllowInvalidCertificates,
```

Comment:
This comment will be public facing, since it's on the public interface, so we probably want to rephrase it in a way that makes sense to a user reading this (and take care of the typo in the `tlsCertificateKeyFilePassword`). Additionally, `tlsAllowInvalidCertificates`, `tlsAllowInvalidCertificates`, and `tlsInsecure` aren't in the table below, so we should document them. 

It might be most user-friendly to update the table heading from `MongoDB equivalent` to `MongoDB driver spec equivalent` and then add a column for `Legacy option name` (or something along those lines) instead of just listing them in the paragraph above, that way it's easy to see what maps to what.

Lastly, `crl` will no longer have an equivalent config option if we deprecate the sslCRL option - are we ok not letting users set it at all, or do we still need to provide *some* way for users to set it even if the drivers spec doesn't account for it?

Reply:
I've update the table per the suggestion. As for `crl`/`sslCRL` - @addaleax @dariakp  if shell feels that we do still need support for it, even though it's not in the spec I think it's fine to not deprecate it. Maybe we can update the deprecation ticket to also add a more "spec compliant looking" name like `tlsCRL` so they all have the same naming convention?